src/MetricsUploader/MetricsUploaderWindows.cpp

```diff
@@ -47,8 +47,9 @@ class MetricsUploaderImpl : public MetricsUploader {
  private:
   [[nodiscard]] bool FillAndSendLogEvent(OrbitLogEvent partial_filled_event) const;
 
+  mutable absl::Mutex mutex_;
   HMODULE metrics_uploader_client_dll_;
-  Result (*send_log_event_addr_)(const uint8_t*, int) = nullptr;
+  Result (*send_log_event_addr_)(const uint8_t*, int) ABSL_PT_GUARDED_BY(mutex_) = nullptr;
```

Comment:
I wouldn't promise that this is guarded by the mutex. I would state in a comment that only the SendLogEvent methods can be called concurrently from different threads. Anything else is not thread-safe.